designer/server/src/main/scala/pl/touk/nussknacker/ui/process/ScenarioActivityAuditLog.scala

```diff
@@ -0,0 +1,221 @@
+package pl.touk.nussknacker.ui.process
+
+import com.typesafe.scalalogging.LazyLogging
+import pl.touk.nussknacker.engine.api.deployment._
+import pl.touk.nussknacker.engine.api.process.{ProcessId, VersionId}
+import pl.touk.nussknacker.ui.process.ScenarioAttachmentService.AttachmentToAdd
+import pl.touk.nussknacker.ui.security.api.LoggedUser
+
+object ScenarioActivityAuditLog extends LazyLogging {
+
+  def onCreateScenarioActivity(
+      scenarioActivity: ScenarioActivity
+  ): Unit =
+    logger.info(
```

Comment:
shouldn't we create a logger with the name we explicitly control? (at the moment it's related to the class name) The right answer to the question will be related to business requirements I think.